packages/react-reconciler/src/ReactFiberCompleteWork.js

```diff
@@ -1108,17 +1095,17 @@ function completeWork(
               );
             }
 
-            // This must come at the very end of the complete phase.
-            bubbleProperties(workInProgress);
             preloadInstanceAndSuspendIfNeeded(
               workInProgress,
               type,
               newProps,
               renderLanes,
             );
-            return null;
           }
         }
+        // This must come at the very end of the complete phase.
+        bubbleProperties(workInProgress);
+        return null;
```

Comment:
Previously bubbleProperties was called before preloadInstance/preloadResource but these functions mutate flags and so I suspenct that was incorrect all along